subsys/bluetooth/controller/hci/hci_internal.h

```diff
@@ -23,18 +23,31 @@ extern atomic_t hci_state_mask;
 #define HCI_CLASS_EVT_CONNECTION  2 /* Connection management; e.g.
 				     * terminate, update, encryption
 				     */
-#define HCI_CLASS_ACL_DATA        3 /* Asynchronous Connection Less (general
+#define HCI_CLASS_EVT_LLCP        3 /* LL Control Procedures */
```

Comment:
Are these numbers not SPEC defined?
If so I would make NONE 0.
Maybe make them an enum? This could give warnings if you don't handle all cases for example.

Reply:
My point was that we use them in switch cases, and the compiler can give you warning when you have don't handle all values of the enum. But no strong opinion, just a suggestion.